roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -168,10 +170,6 @@
   include: kubeadm-secondary-legacy.yml
   when: not kubeadm_control_plane
 
-- name: kubeadm | upgrade kubernetes cluster
```

Comment:
why is this disabled?

Reply:
Because if upgrade kubernetes version, kubernetesVersion in file kubeadm-config.yaml will be changed, `kubeadm_config_changed` is true, then it will re-run kubeadm init.

Reply:
kubeadm upgrade not work because it start CoreDNS, and kubeadm upgrade do not support --skip-phases=addon/coredns like kubeadm init, so re-run kubeadm init instead of kubeadm upgrade